bundle/edu.gemini.qpt.client/src/main/scala/edu/gemini/qpt/TestLauncher.scala

```diff
@@ -0,0 +1,51 @@
+package edu.gemini.qpt
+
+import java.io.File
+
+import edu.gemini.ags.conf.ProbeLimitsTable
+import edu.gemini.qpt.ui.ShellAdvisor
+import edu.gemini.qpt.ui.action.PublishAction
+import edu.gemini.spModel.core.{Peer, Site, Version}
+import edu.gemini.ui.workspace.impl.Workspace
+import edu.gemini.util.trpc.auth.TrpcKeyChain
+import edu.gemini.util.security.auth.keychain.Action._
+import edu.gemini.util.security.auth.ui.AuthDialog
+
+/** Standalone launcher for running QPT inside an IDE. */
+object QptTestLauncher {
```

Comment:
No, in general you don't want to use `App` because it has weird initialization semantics. Just use an object than implements `main`.